.ci/docker/check-docker-compose-services.sh

```diff
@@ -16,14 +16,15 @@ is_healthy() {
         echo "$1 is healthy"
         return 0
     else
-        echo "Waiting for $1"
+        echo "Waiting for $1, logs tail is:"
+        docker-compose logs --tail=10 ${service}
         return 1
     fi
 }
 
 # on CI we want to test ALL services (discovery, config and gateway) using dev profile (instead of the local default)
 docker-compose pull -q
-docker-compose up --no-build -d gateway-server config-server discovery-server
+docker-compose up --no-build -d discovery-server config-server gateway-server
```

Comment:
whats the difference

Reply:
I thought to order it by importance that might cause the discovery to come online first (there might be priority/resourcing consideration)